SOLIDHomework.Core/Services/OrderService.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using SOLIDHomework.Core.Interfaces;
+using SOLIDHomework.Core.Model;
+using SOLIDHomework.Core.Payment;
+using SOLIDHomework.Core.Services;
+
+namespace SOLIDHomework.Core
+{
+    //Order - check inventory, charge money for credit card and online payments, 
+    //tips:
+    //think about SRP, DI, OCP
+    //maybe for each type of payment type make sense to have own Order-based class?
+    public class OrderService
```

Comment:
Add abstraction layer for OrderService in order to follow Dependency Inversion principle and to work with DI container

Reply:
interface for OrderService